tests/database/migrations/2014_10_12_000000_create_quicksand_people_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
Is this marked in a way that it won't get migrated on their actual app?

Reply:
@mattstauffer Yes, if you wanted to run migrations on the users actual app, you'd have to call `loadMigrationsFrom()` inside the `boot` method of your packages service provider. Since we don't do that, these won't get run for the user (we just run them in our tests). I tested this manually and confirmed the migrations don't run.

Relevant orchestral documentation: https://github.com/orchestral/testbench#running-testing-migrations

Reply:
@imjohnbon thought it was something like that. Thanks!